[EDU-6683] fix: refactor Edge App main settings reference with v4 changes (#1802)
aziontech/docs: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/v3/main-settings-v3.mdx

---
title: Main Settings | v3
description: >-
  Personalize a entrega de sua aplicação com a variedade de opções de protocolo
  da Azion, incluindo suporte para HTTP, HTTPS e HTTP/3 para uma melhor
  experiência do usuário e proteja a conexão com cifras TLS.
meta_tags: >-

>   delivery, ports, http, http3, http2, protocol, tls, encryption, protocolo,

Reviewer: https?

  porta, entrega
namespace: documentation_products_edge_application_delivery_v3
permalink: /documentacao/produtos/build/edge-application/main-settings/v3/
---
import Tag from 'primevue/tag'
import LinkButton from 'azion-webkit/linkbutton'

import Apiv4Rollout from '~/includes/snippets/apiv4Rollout/en/snippet.mdx'

<Apiv4Rollout />

:::caution[importante]
Você está visualizando uma versão anterior desta documentação. Se a sua conta já foi migrada, consulte a [referência de main settings Edge Application](/pt-br/documentacao/produtos/build/edge-application/main-settings/) atualizada.
:::

Com [Edge Application](/pt-br/documentacao/produtos/build/edge-application/), você pode escolher o protocolo de entrega e as portas de sua aplicação. Você também pode garantir a segurança da conexão HTTPS entre a aplicação e seus usuários com cifras TLS.

---

## Protocolos de entrega

Você pode personalizar a entrega de sua aplicação através das seguintes opções:

- **HTTP**: entrega sua aplicação usando apenas o protocolo HTTP.
- **HTTP & HTTPS**: entrega sua aplicação usando os protocolos HTTP e HTTPS.

:::note

> Se você escolher entregar sua aplicação usando HTTP & HTTPS, você deve associar um [digital certificate](/pt-br/documentacao/produtos/secure/edge-firewall/digital-certificates/) a seu domínio.

Reviewer: ```suggestion
Se você escolher entregar sua aplicação usando HTTP e HTTPS, você deve associar um [digital certificate](/pt-br/documentacao/produtos/secure/edge-firewall/digital-certificates/) a seu domínio.
```

:::

### Suporte a HTTP/3

> Se você escolher entregar sua aplicação através de **HTTP & HTTPS**, você pode habilitar o **HTTP/3 support**. Baseado no protocolo QUIC, HTTP/3 proporciona menores tempos de carregamento e latência quando comparados a versões anteriores.

Reviewer: ```suggestion
Se você escolher entregar sua aplicação através de **HTTP e HTTPS**, você pode habilitar o **suporte a HTTP/3**. Baseado no protocolo QUIC, HTTP/3 proporciona menores tempos de carregamento e latência quando comparados a versões anteriores.
```

Ao habilitar suporte a HTTP/3, suas edge applications podem utilizar essa versão do protocolo em navegadores compatíveis somente através da porta *80 em HTTP* e porta *443 em HTTPS*.

Na primeira requisição de um usuário a uma edge application com HTTP/3, o handshake e a primeira resposta serão conduzidos usando TCP e HTTP/1.1 ou HTTP/2. A resposta dessa troca atribui um valor ao [cabeçalho Alt-Svc](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Alt-Svc) que indica que a versão mais recente do protocolo está disponível para o navegador. Se o navegador for compatível com HTTP/3, o protocolo QUIC e o HTTP/3 serão usados, a menos que a resposta em cache falhe ou expire.

---

## Portas

> A Azion permite que você selecione quais portas HTTP e HTTPS serão usadas pela sua aplicação. Você deve selecionar *pelo menos uma porta* das portas disponíveis para cada protocolo escolhido, mas você pode selecionar dentre quaisquer portas disponíveis para entrega.

Reviewer: Sugestão mais enxuta, daí teria que ver se aparece em mais alguma doc
```suggestion
A Azion permite que você configure as portas HTTP e HTTPS para sua aplicação. Para cada protocolo selecionado, você deve escolher pelo menos uma porta dentre as opções disponíveis para entrega.
```

:::note
Se você habilitar [HTTP/3 support](#http3-support), a configuração multiport será sobrescrita para as portas padrão: 80 em HTTP e 443 em HTTPS. Multiport simultâneo não está disponível para aplicações com suporte a HTTP/3.
:::

#### Lista de portas disponíveis

> | HTTP Port    | HTTPS Port    |

Reviewer: ```suggestion
| Porta HTTP    | Porta HTTPS    |
```

|--------------|---------------|
| 80 (padrão)  | 443 (padrão)  |
| 8008         | 8443          |
| 8080         | 9440          |
| 8880         | 9441          |
|              | 9442          |
|              | 9443          |
|              | 7777          |
|              | 8888          |
|              | 9553          |
|              | 9653          |
|              | 8035          |
|              | 8090          |

---

## Versão mínima de TLS

O protocolo **Transport Layer Security (TLS)** permite que você criptografe o tráfego da web. As versões de TLS que podem ser usadas com edge applications são:

> - TLS 1.0 (*descontinuado*)
> - TLS 1.1 (*descontinuado*)

Reviewer: Aqui entendo que são AS versões
```suggestion
- TLS 1.0 (*descontinuada*)
- TLS 1.1 (*descontinuada*)
```

- TLS 1.2
- TLS 1.3

Você poderá escolher a versão mínima de TLS que será suportada para criptografar o tráfego de sua edge application. Ao escolher versões recentes do protocolo, dispositivos ou navegadores mais antigos podem não conseguir acessar a edge application.

> Por padrão, a Azion bloqueia *TLS Renegotiation* e *TLS Resumption*. Se você deseja personalizar essa configuração, entre em contato com nossa [equipe de Vendas](https://www.azion.com/pt-br/contate-vendas/).

Reviewer: /pt-br/contate-vendas/?

| ECDHE-RSA-AES128-GCM-SHA256   | ✔︎                 | ✔︎                 | ✔︎                 | ✔︎               |
| ECDHE-RSA-AES256-GCM-SHA384   | ✔︎                 | ✔︎                 | ✔︎                 | ✔︎               |
| ECDHE-RSA-CHACHA20-POLY1305   | ✔︎                 | ✔︎                 | ✔︎                 | ✔︎               |
| ECDHE-ECDSA-AES256-SHA384     | ✔︎                 | ✔︎                 | ✕                 | ✕               |
| ECDHE-ECDSA-AES128-SHA256     | ✔︎                 | ✔︎                 | ✕                 | ✕               |
| ECDHE-RSA-AES128-SHA256       | ✔︎                 | ✔︎                 | ✕                 | ✕               |
| ECDHE-RSA-AES256-SHA384       | ✔︎                 | ✔︎                 | ✕                 | ✕               |
| AES128-SHA256                 | ✔︎                 | ✕                 | ✕                 | ✕               |
| AES256-GCM-SHA384             | ✔︎                 | ✕                 | ✕                 | ✕               |
| AES128-GCM-SHA256             | ✔︎                 | ✕                 | ✕                 | ✕               |

> <LinkButton link="/pt-br/documentacao/produtos/guias/secure/cifras/" label="consulte o guia configurar o conjunto de cifras TLS" severity="secondary" />

Reviewer: Usei o nome do guia mostrado no link
```suggestion
<LinkButton link="/pt-br/documentacao/produtos/guias/secure/cifras/" label="consulte o guia Como configurar o conjunto de cifras TLS para edge applications" severity="secondary" />
```
